client.js

```diff
@@ -4,122 +4,203 @@ const JsonSocket = require('json-socket');
 const port = 5016;
 const host = '127.0.0.1';
 
-const clientsNumber = parseInt(process.argv[2]) || 1; // reading the command line argument
-let clientsLeft = clientsNumber;
-
-function makeConnection(clientId) {
+/**
+ * Creates a simple submission with Main.java class and Main.main method
+ * @param command : command to insert in the main method
+ * @return {{main: 'Main.java', files: [{name: 'Main.java', data: String}]}}
+ */
+function simpleSubmission(command) {
+
+    const result = {
+        main: 'Main.java',
+        files: [{
+            name: 'Main.java',
+            data: `
+        import java.util.*;
```

Comment:
wouldn't it be more clean and easy to maintain and test in isolation to have the java strings in external files?